rules/linux/execution_abnormal_process_id_file_created.toml

```diff
@@ -74,19 +74,20 @@ references = [
 risk_score = 47
 rule_id = "cac91072-d165-11ec-a764-f661ea17fbce"
 severity = "medium"
-tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Execution", "Threat: BPFDoor", "Resources: Investigation Guide", "Data Source: Elastic Endgame"]
+tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Execution", "Threat: BPFDoor", "Resources: Investigation Guide", "Data Source: Elastic Defend"]
 timestamp_override = "event.ingested"
 type = "new_terms"
 
 query = '''
 host.os.type : "linux" and event.category : "file" and event.action : ("creation" or "file_create_event") and
```

Comment:
```suggestion
host.os.type : "linux" and event.category : "file" and event.action == "creation" and
```

Reply:
Good catch, but it's KQL rather than EQL. Will fix it! Thanks

Reply:
>but it's KQL rather than EQL

Oops